java-extensions/hudi-reader/src/main/java/com/starrocks/hudi/reader/HudiColumnValue.java

```diff
@@ -159,4 +149,22 @@ public byte getByte() {
     public BigDecimal getDecimal() {
         return ((HiveDecimal) inspectObject()).bigDecimalValue();
     }
+
+    @Override
+    public LocalDate getDate() {
+        return LocalDate.parse(inspectObject().toString(), DATE_FORMATTER);
+    }
+
+    @Override
+    public LocalDateTime getDateTime(ColumnType.TypeValue type) {
+        // INT64 timestamp type
+        if (HudiScannerUtils.isMaybeInt64Timestamp(type) && (fieldData instanceof LongWritable)) {
+            long datetime = ((LongWritable) fieldData).get();
+            TimeUnit timeUnit = TIMESTAMP_UNIT_MAPPING.get(type);
+            LocalDateTime localDateTime = HudiScannerUtils.getTimestamp(datetime, timeUnit, true);
```

Comment:
remove `formatDateTime` method and move `XXX_FORMATTER` in `HudiScannerUtils` to this class